package/src/components/SelectableList/v1/SelectableList.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { withComponents } from "@reactioncommerce/components-context";
+import { CustomPropTypes, applyTheme } from "../../../utils";
+
+const StyledList = styled.div`
+  width: 100%;
+  fieldset {
+    border-top: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+    border-left: none;
+    border-right: none;
+    border-bottom: none;
+    border-top-right-radius: ${applyTheme("selectableListBorderRadius")};
+    border-top-left-radius: ${applyTheme("selectableListBorderRadius")};
+    padding: ${applyTheme("selectableListPadding")};
+    margin: ${applyTheme("selectableListMargin")};
+    .wrapper {
+      border-bottom: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+      border-left: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+      border-right: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+      padding: ${applyTheme("selectableListItemPadding")};
+      .leftAligned {
+        justify-content: flex-start;
+        label {
+          font-weight: ${applyTheme("selectableListLabelFontWeight")}
+        }
+        .detail {
+          font-size: ${applyTheme("selectableListDetailFontSize")};
+          font-family: ${applyTheme("selectableListFontFamily")};
+          margin-left: 2px;
+        }
+      }
+      .amex,
+      .visa {
```

Comment:
I don't like having amex and visa classes in here. Can you make this thing unaware of what is being rendered within it?

And in general I don't think we should be adding `className` to anything or using classes as style selectors. That is a DOM-centric approach which would likely need to be rewritten to use these in React Native some day. You should instead make a `WrapperDiv = styled.div` and use that.